README.md

```diff
@@ -1,156 +1,7 @@
-
-Sequences which are not at least 75% similar by sequence identity to any record in the tree to insert into are not inserted into the tree.
-
-A fragment may be reasonable to insert into multiple locations. However, downstream methods such as UniFrac cannot currently handle placement distributions. As such, the placement with the highest likelihood is choosen.
-
-## Files produced
-
-The plugin will generate two files:
-  1. A `Phylogeny[Rooted]` type: This is the tree with the sequences placed (which could be inserted), and are identified by their corresponding sequence IDs. You can directly use this tree for phylogenetic diversity computation like UniFrac or Faith's Phylogenetic Diversity.
-  2. A `Placements` type: It is a JSON object which, for every input sequence, describes the different possible placements.
-
-## Example
-
-QIIME 2's "Moving Pictures" [tutorial](https://docs.qiime2.org/2017.10/tutorials/moving-pictures/#generate-a-tree-for-phylogenetic-diversity-analyses) suggests constructing a de-novo phylogeny for the fragments, i.e `FeatureData[Sequence]`, to obtain a `Phylogeny[Rooted]` that can be used for phylogenetic diversity computation. "Fragment insertion" via this plugin provides an alternative way to acquire the `Phylogeny[Rooted]` by inserting sequences of `FeatureData[Sequence]` into a high quality reference phylogeny and thus provides multiple advantages over de-novo phylogenies, e.g. accurate branch lengths, multi-study meta-analyses, mixed region meta-analyses (e.g. V4 and V2).
-
-Let us use the `FeatureData[Sequence]` from QIIME 2's tutorial as our input:
-
-   - `rep-seqs.qza`: [view](https://view.qiime2.org/?src=https%3A%2F%2Fdocs.qiime2.org%2F2017.10%2Fdata%2Ftutorials%2Fmoving-pictures%2Frep-seqs.qza) | [download](https://docs.qiime2.org/2017.10/data/tutorials/moving-pictures/rep-seqs.qza)
-
-The following single command will produce two outputs: 1) `phylogeny.qza` is the `Phylogeny[Rooted]` and 2) `placements.qza` provides placement distributions for the fragments (you will most likely ignore this output) (Computation might take some 10 minutes):
-```
-qiime fragment-insertion sepp \
-  --i-representative-sequences rep-seqs.qza \
-  --o-tree insertion-tree.qza \
-  --o-placements insertion-placements.qza
-```
-Output artifacts:
-   - `insertion-tree.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/blob/master/Example/insertion-tree.qza?raw=true)
-   - `insertion-placements.qza`: ~[view]()~ | [download](https://github.com/biocore/q2-fragment-insertion/blob/master/Example/insertion-placements.qza?raw=true)
+# q2-fragment-insertion
```

Comment:
why did you remove the readme content and where can user now find this information? Many people asked e.g. for runtime estimations and this document was my collection of answers to those common questions.

Reply:
This information will be moved to library.qiime2.org